packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.controller.js

```diff
@@ -29,7 +41,43 @@ export default class TelecomPackMigrationCtrl {
           this.process.migrationTaskId = head(pendingTasks);
           this.process.migrationDoing = true;
         } else {
-          this.process.currentStep = 'offers';
```

Comment:
You should keep this line, remove `else` conditions below, and just keep the code at L55